Utilities/Distribution/MacARM/build_all.sh

```diff
@@ -0,0 +1,56 @@
+#! /usr/bin/env bash
+
+SIMPLEITK_GIT_TAG='v2.1rc2'
```

Comment:
I have used a line similar to `SIMPLEITK_GIT_TAG=${SIMPLEITK_GIT_TAG:-v1.1rc1}` for this purpose. If the value is set in the environment it uses it, otherwise there is a default.

Reply:
Yeah, I just hard coded it because I'm not really a bash guy, and didn't feel like doing proper command line parsing.